jctools-core/src/main/java/org/jctools/queues/ConcurrentCircularArrayQueue.java

```diff
@@ -24,12 +24,11 @@
 }
 
 /**
- * A concurrent access enabling class used by circular array based queues this class exposes an offset computation
- * method along with differently memory fenced load/store methods into the underlying array. The class is pre-padded and
- * the array is padded on either side to help with False sharing prvention. It is expected theat subclasses handle post
- * padding.
+ * A concurrent access enabling base class for circular array queues. This class exposes an offset computation
```

Comment:
This is a welcome cleaup, but formatting/spelling etc are easier to accept as separate PRs.